frontend/src/container/ListAlertRules/ListAlert.tsx

```diff
@@ -128,6 +158,13 @@ function ListAlert({ allAlertRules, refetch }: ListAlertProps): JSX.Element {
 		}
 	};
 
+	const handleSearch = useDebouncedFn((e: unknown) => {
+		setSearchString((e as React.BaseSyntheticEvent).target.value);
```

Comment:
nit pick, can assign e.target.value to a variable and use in both lines.

Reply:
Done.